Source/Mixers/iOS/GLESVideoMixer.swift

```diff
@@ -266,10 +118,10 @@ open class GLESVideoMixer: IVideoMixer {
             }
             
             self.glesCtx = nil
```

Comment:
Same as [this](https://github.com/openfresh/VideoCast-Swift/pull/1/files#r181368490) goes for here.